aleth/MinerAux.h

```diff
@@ -30,16 +30,7 @@
 using namespace std;
```

Comment:
Could you also please get rid of the rest of `using namespace` in this header, it shouldn't be difficult